src/Components/HumanAtNasa/HumanAtNasa.tsx

```diff
@@ -14,7 +14,7 @@ export function HumanAtNasa(props : HumanAtNasaProps){
         <div className='humansComponent'>
             <img className='humansImage' src={props.imageUrl}/>
             <h2 className='humanNames'>{props.name}</h2>
-            <p className='jobTitle'>{props.jobTitle}</p>
+            <h3 className='jobTitle'>{props.jobTitle}</h3>
```

Comment:
Is this actually a heading?

Reply:
Hi Mike, yes, this should be a subheading :)